influx-writer/influx-writer.py

```diff
@@ -36,6 +36,12 @@ def __init__(self, ls, influx, redis):
         self.inserts = 0
         self.time = 0
 
+    @staticmethod
```

Comment:
For now, that is fine. Ideally I would like to get rid of all python modules (not scripts) from root folder... But we can tackle that later.